### PR TITLE
feat(meeting-containers): decrypt function

### PR DESCRIPTION
**Pull request by yjlo**
# COMPLETES
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-315399
## This pull request addresses
Being able to decrypt meeting containers

## by making the following changes
Adding a new function to decrypt transforms

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js [hunk 1]
```diff
@@ -138,6 +138,32 @@ export const transforms = toArray('inbound', {
...
+
+    if (meetingContainerActivity.displayName && meetingContainerActivity.extensions) {
+      promises.push(ctx.transform('decryptPropDisplayName', key, meetingContainerActivity));
+      const itemsToDecrypt = meetingContainerActivity.extensions.items.filter((item) => item.data.objectType === 'recording');
+
+      itemsToDecrypt.forEach((itemToDecrypt) => {
+        promises.push(ctx.transform('decryptPropTopic', itemToDecrypt.encryptionKeyUrl, itemToDecrypt.data));
+      });
+
+
```
Comment (chburket) [inline packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js:159]: nit: remove extra empty lines

**Review by chburket — CHANGES_REQUESTED** [review]
At the very least, need tests in the test function. Check the other comments, too.

File: packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js [hunk 2]
```diff
@@ -138,6 +138,32 @@ export const transforms = toArray('inbound', {
...
   },
 
+  /**
+ * Decrypt an individual meeting container activity
+ * @param   {Object} ctx      An object containg a webex instance and a transform
+ * @param   {object} key      hello
+ * @param   {Object} meetingContainerActivity An instance of a Webex meeting container activity
+ * @returns {Promise}         Returns a ctx.transform promise
+ */
+  decryptMeetingcontainer(ctx, key, meetingContainerActivity) {
```
[the remark this answers is not in the document]
Comment (yjlo) [reply packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js]: My bad spelling and typo is at fault

File: packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js [hunk 3]
```diff
@@ -138,6 +138,32 @@ export const transforms = toArray('inbound', {
...
+    const promises = [];
+
+    if (meetingContainerActivity.displayName && meetingContainerActivity.extensions) {
+      promises.push(ctx.transform('decryptPropDisplayName', key, meetingContainerActivity));
```
[the remark this answers is not in the document]
Comment (yjlo) [reply packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js:152]: I agree making that change now

**Pull request closed without merging**